backend/db/ad_hoc_omop_vocab_version.py

```diff
@@ -0,0 +1,58 @@
+"""Ad hoc script: to populate omopVocabVersion in the database
```

Comment:
Ad hoc script: to populate `omopVocabVersion` in the database

Since this only needs to be used one time, should we delete the script or keep it?